Multisig/UI/Settings/OwnerKeyManagement/Backup/UIFlow.swift

```diff
@@ -71,13 +75,8 @@ class UIFlow: NSObject {
     }
 
     func stop(success: Bool) {
-        if let presenter = presenter, presenter.presentedViewController != nil {
-            presenter.dismiss(animated: true) { [unowned self] in
-                completion(success)
-            }
-        } else if let presenter = navigationController.presentingViewController {
-            // in this case, the presenting view controller is not a direct ancestor of the
-            // currently opened view controller
+        if let presenter = presenter {
+            assert(presenter.presentedViewController != nil)
```

Comment:
I would replace assert by optional unwrapping, just to not get a crash in future